### ISSUE TITLE
Microservices For Hindi.

### ISSUE THREAD

**Issue by gauravrattan** [cncf/glossary#1572]
Hindi version of Microservices is not available.
Link=https://glossary.cncf.io/microservices/

**Comment by akshat5302** [comment]
Hey, @gauravrattan I saw that you had opened up an issue regarding cloud computing earlier as well. Do follow contributing guidelines of cncf glossary and do not open another issue until the previously opened issue got merged 
It's just a friendly reminder to work on one issue at a time 

https://glossary.cncf.io/contribute/#propose-new-terms

**Comment by gauravrattan** [comment]
@akshat5302  ok brother

**Comment by abhijeet-26** [comment]
I would like to work on this issue. Please assign me on this issue

**Comment by devarsh10** [comment]
I had translated it.
Here's the first translated paragraph of it.

Microservices एक नया तरीका है Application Development के लिए जो cloud-native Technologies का उपयोग करती है | 

जैसे कि Netfilx[Application] जो सिर्फ एक एप्लीकेशन दिखती है किंतु बहुत सारी छोटी-छोटी सर्विसिस को इकट्ठा करके काम करती है| 

जैसे सर्च करने के लिए अलग सर्विस,  प्रीव्यू करने के लिए अलग सर्विस, और आदि| 

तात्पर्य, Microservices, Monolithic Applications के विपरीत है |

Will it do?

Regards,
Devarsh.

**Comment by akshat5302** [comment]
Hey don't be in hurry @devarsh10 read the contributing guidelines first
https://glossary.cncf.io/contribute/#propose-new-terms
You are doing it all wrong and not following the steps of contributing guidelines

**Comment by devarsh10** [comment]
Okay.

**Comment by chirag0002** [comment]
Hey @akshat5302, I have raised a PR resolving this following issue, following the contributing guidelines.
Can you review it!!

**Comment by akshat5302** [comment]
I'm not an official reviewer you can check reviewers, in the pending reviewer's section and tag them. Well, there is no need to tag them as well cause one request has already been sent to reviewers to check your PR and they will soon contact you regarding the same 
Regards 
Akshat Jain

**Comment by Garima-Negi** [comment]
duplicate of [https://github.com/cncf/glossary/issues/728](https://github.com/cncf/glossary/issues/728)
